lib/server/env.js

```diff
@@ -58,6 +61,12 @@ class Versioning {
         ? siteConfig.defaultVersionShown
         : this.versions[0]; // otherwise show the latest version (other than next/master)
     }
+
+    if (!fs.existsSync(versions_js)) {
+      this.error = `${chalk.yellow(
```

Comment:
Please split the error flag and messages into separate fields:

```js
constructor {
  ...
  this.missingVersionsPage = false;
}

printMissingVersionsPageError() {
  console.error(`${chalk.yellow(...)`);
}
```

Reply:
done